modules/mono/glue/cs_files/Quat.cs

```diff
@@ -192,22 +199,56 @@ public Vector3 Xform(Vector3 v)
             return new Vector3(q.x, q.y, q.z);
         }
 
+        // Static Readonly Properties
+        public static Quat Identity { get; } = new Quat(0f, 0f, 0f, 1f);
+
         // Constructors 
         public Quat(real_t x, real_t y, real_t z, real_t w)
         {
             this.x = x;
             this.y = y;
             this.z = z;
             this.w = w;
-        }   
+        }
+
+        public bool IsNormalized()
+        {
+            return Mathf.Abs(LengthSquared - 1) <= Mathf.Epsilon;
```

Comment:
Oops, reviewed old commit. Shouldn't this be Length instead of LengthSquared?

Reply:
Done!